packages/gem/sentry-opentelemetry/5.7.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Sentry OpenTelemetry SDK",
+  "canonical": "gem:sentry-opentelemetry",
+  "version": "5.7.0",
+  "package_url": "https://rubygems.org/gems/sentry-opentelemetry",
+  "repo_url": "https://github.com/getsentry/sentry-ruby",
+  "main_docs_url": "https://docs.sentry.io/platforms/node/performance/instrumentation/opentelemetry/"
```

Comment:
This links to node. Was this on purpose?

Reply:
forgot to s/, thx